collections/concepts/access-controls.md

```diff
@@ -12,16 +12,266 @@ width: 1200
 height: 630
 ---
 
-# What are Access Controls?
+# Access Controls
+
+Basis Theory secures sensitive data as [Tokens](/concepts/what-are-tokens) that are stored within an isolated 
+compliant environment, called a [Tenant](/concepts/access-controls#what-are-tenants).
+Your systems interact with Tokens or Services through the Basis Theory API using an API key that is issued to 
+an [Application](/concepts/access-controls#what-are-applications).
+Access to your Tenant's data is limited by applying access controls to an Application via 
+[Permissions](https://docs.basistheory.com/api-reference/#permissions) and [Access Rules](/concepts/access-controls#what-are-access-rules)
+to ensure services and users only have access to the data they need.
+
+
+## What are Tenants?
+
+Everything starts with a Tenant in your Basis Theory account. Tenants represent an isolated environment in which your
+[Tokens](/concepts/what-are-tokens) and other resources are defined, managed, and stored.
+Data is not shared between Tenants, but you’re allowed to create as many Tenants as you’d like within Basis Theory.
+You can use multiple Tenants to isolate different domains of data, and to support your Software Development Life Cycle (SDLC).
+
+The user account that was used to create a Tenant in the [Portal](https://portal.basistheory.com/members)
+will be designated as the Tenant Owner. New members can be invited to an existing Tenant through the "Members" tab.
+Managing Tenant Members can also be done directly through the [API](https://docs.basistheory.com/#tenant-members).
+
+### Common Use Cases for Tenants
+
+#### Enabling Your Software Development Life Cycle
```

Comment:
did Rad see how the use cases titles look like? I don't like them

Reply:
previously it looked better as an h3, now it is an h4 and looks bad in my opinion, but I see why you had to do it. We should talk to Rad or Rafa

Reply:
We discussed and I'm changing the style for h4 elements to look like h3 with a slightly smaller font size.